docs/mp/integrations/wls.adoc

```diff
@@ -0,0 +1,392 @@
+
+///////////////////////////////////////////////////////////////////////////////
+
+
+= Oracle WebLogic Server 
+:description: Helidon Oracle WebLogic Server Integration
+:keywords: wls, jms, rest, single sign-on, sso, microtx, web services
+:feature-name: Oracle WebLogic Server Integration Integration
+:rootdir: {docdir}/../..
+
+include::{rootdir}/includes/mp.adoc[]
+
+== Contents
+
+- <<Overview, Overview>>
+- <<REST Services Integration, REST Services Integration>>
+- <<JMS Integration, JMS Integration>>
+- <<Web Services Integration, Web Services Integration>>
+- <<Single Sign-On on OCI Integration, Single Sign-On on OCI Integration>>
+
+== Overview
+
+The Oracle WebLogic Server and Helidon integration enables interaction between a Helidon microservice application and an application installed on Oracle WebLogic Server, which is deployed in a Kubernetes cluster managed by the WebLogic Kubernetes Operator (WKO).
+
+The following integrations are available for your use:
+
+* The REST integration allows Oracle WebLogic Server applications and Helidon microservices to communicate through RESTful web service invocations, in one or both directions.
+* The JMS integration allows Oracle WebLogic Server applications and Helidon microservices to communicate through messaging, in one or both directions.
+* The Oracle WebLogic Web Services integration allows Helidon microservices to interact with Oracle WebLogic Server applications through SOAP (Simple Object Access Protocol) web service calls from Helidon to Oracle WebLogic Server.
+* The Oracle WebLogic Server and Helidon integration allows communication between a WebLogic cluster-hosted application and a Helidon microservice application by implementing Single Sign-on (SSO) authentication using Oracle Identity Cloud Service (IDCS).
+
+
+== REST Services Integration [[REST-Services]]
+
+- <<REST-Services-Overview, Overview>>
+- <<REST-Services-Maven-Coordinates, Maven Coordinates>>
+- <<REST-Services-Usage, Usage>>
+- <<REST-Services-References, References>>
+
+=== Overview [[REST-Services-Overview]]
+The REST services integration with Helidon enables bidirectional REST calls between Helidon and Oracle WebLogic Server. The Oracle WebLogic Server applications and Helidon microservices communicate through RESTful Web Service invocations, in one or both directions.
+
+=== Maven Coordinates [[REST-Services-Maven-Coordinates]]
+
+To enable Oracle WebLogic Server REST Integration, add the following dependencies to your project’s `pom.xml` file:
+
+[source,]
+----
+<dependency>
+    <groupId>io.helidon.microprofile.bundles</groupId>
+    <artifactId>helidon-microprofile-core</artifactId>
+</dependency>
+<dependency>
+    <groupId>com.fasterxml.jackson.core</groupId>
+    <artifactId>jackson-databind</artifactId>
+    <version>2.14.1</version>
+</dependency>
+<dependency>
+    <groupId>org.junit.jupiter</groupId>
+    <artifactId>junit-jupiter-api</artifactId>
+    <scope>test</scope>
+</dependency>
```

Comment:
Why Jackson? Why Jupiter?

Reply:
Agree. We should embrace JSON-B instead of Jackson. Jupiter is for testing only and users may not need it at all.